tasmota/tasmota_xsns_sensor/xsns_62_esp32_mi_ble.ino

```diff
@@ -2711,7 +2711,7 @@ void CmndMi32Keys(void){
           return;
         }
 
-        AddLog(LOG_LEVEL_ERROR,PSTR("M32: Add key mac %s = key %s"), mac, key);
+        AddLog(LOG_LEVEL_INFO, PSTR("M32: Add key mac %s = key %s"), mac, key);
```

Comment:
This change does not belong in the MR.

Reply:
Found while testing, so fixed it directly.

Reply:
What's the reason for changing the log level?

Reply:
Adding a BLE key is not an error, just an info.